src/pages/epic-spells.tsx

```diff
@@ -12,6 +13,18 @@ export const EpicSpells = () => {
                 </div>
             </header>
             <section>Create an epic spell.</section>
+            <div>
+                <ul>
+                    {SPELL_SEEDS.map(seed => (
+                        <div key={seed.id}>
+                            <span>
+                                <input type="checkbox" id={seed.id.toString()} name={seed.name + '-checkbox'}></input>
```

Comment:
> `id={seed.id.toString()`

Probably not needed to convert the number to a string here.

> name={seed.name + '-checkbox'}

I would also use a [string template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) for the name just for cleanliness:

```
name={`${seed.name}-checkbox`}
```

Reply:
I was trying to figure out the string template literal and just couldn't make it work. I didn't have enough '{' in there.

I thought I didn't need to make the id a string either, but the id property on an input only takes a string and the id on the model is a number. I'll just make the id a string if that makes more sense. I didn't have a specific reason for making it a number to begin with.

Reply:
ae4602f